configure-linux.sh

```diff
@@ -30,6 +30,10 @@ function HELP {
    echo ""
    echo "--set-direction-enabled  --Set direction for capturing incoming or outgoing packets (supported on libpcap>=0.9.1)"
    echo ""
+   echo "--asan                   --Build PcapPlusPlus with Address Sanitizer enabled"
```

Comment:
it'd be helpful to explain that this switch is related to fuzzing and what's the difference between this flag and the `--fuzzing` flag